src/scrape_up/flipkart/flipkart_file.py

```diff
@@ -456,6 +457,8 @@ def printers(self):
 
         except Exception as e:
             return None
+        
+    def vrbox():
```

Comment:
What is this? you defined the `vrbox()` function on line 460 with no follow on, how you even made the PR, the code is full of errors, and even other endpoints are broken.

Review your code properly before merging the PR.